UnitTests/Bookmarks/Model/LocalBookmarkManagerTests.swift

```diff
@@ -348,6 +348,65 @@ final class LocalBookmarkManagerTests: XCTestCase {
         XCTAssertTrue(resultsWithNotCapitalizedQuery.count == 2)
     }
 
+    @MainActor
```

Comment:
What's the reason these tests are running on the main actor? If you are forced to do this from one of the used objects generally it's a sign of a bad interface.
As a general rule unit tests should never run on main thread

Reply:
@federicocappelli The reason is the `FaviconManagement` interface is defined in the `MainActor`. Changing this is out of the scope of this task, it’s part of the design, that was done a long time ago.

Reply:
Understandable